### PR TITLE
Refactor/ss4

### PR DESCRIPTION
**Pull request by jsirish**

### CODE REVIEW

**Comment by mak001** [comment]
composer dependencies need to be updated

File: src/Model/MailChimpList.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    );
+
+    private static $has_many = array(
+        'MailingSegments' => MailChimpSegment::class,
+    );
+
+    private static $indexes = array(
+        'MCID' => array(
+            'type' => 'unique',
+            //'value' => 'MCID',
```
Comment (jsirish) [inline src/Model/MailChimpList.php]: @mak001 I had issues with the index value so I commented out, could you take a look?

File: src/Model/MailChimpSegment.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    );
+
+    private static $has_one = array(
+        'MailingList' => MailChimpList::class,
+    );
+
+    private static $indexes = array(
+        'MCID' => array(
+            'type' => 'unique',
+            //'value' => 'MCID',
```
Comment (jsirish) [inline src/Model/MailChimpSegment.php]: @mak001 same issue with the index value as above

**Comment by mak001** [comment]
i fixed the index issues

**Review by mak001 — APPROVED** [review]

**Pull request merged**